package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build && npm run build:rss && next export",
-    "build:rss": "node ./scripts/generate-rss-feed.js",
+    "build:rss": "tsc --outDir dist --noEmit false && sed -i 's/markdown\"/markdown.js\"/g' dist/lib/api.js && node scripts/generate-rss-feed.js",
```

Comment:
This is a small workaround for Webpack not supporting ".js" imports in TypeScript source (TypeScript itself does, but Webpack is the issue here).

Since node with `type: module` requires full file extensions, I need to rewrite an import in the compiled javascript code.

An alternative would be using something like https://www.npmjs.com/package/tsx or https://www.npmjs.com/package/ts-node to execute TypeScript directly, but I didn't want to add any new dependencies

Reply:
Do the `npm install` and `npm run dev` commands in the readme still work?

Reply:
Yeah, nothing has changed there. `npm run build:rss` also works as expected